app/Http/Controllers/Store/PaymentController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Store;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+class PaymentController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('guest:store');
+    }
+
+    /** Get a specific payment request by link
+     * @param $paymentUuid
+     * @return mixed
```

Comment:
Are these placeholders? Or will it be the uuid that gets returned?

Reply:
Placeholders, enough to make the route tests work.